### DISCUSSION TITLE
How  can  I stop the account subscription?

### DISCUSSION THREAD

**Discussion by netrasTian** [community/community#127486] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

How  can  I stop the account subscription? I want to stop the yaerly subscription,but I can't get the entrance on the github website

**Comment by netrasTian** [comment]
I don't want to pay for it next year,but I can't do it!!!!

**Comment by a6uxa4** [comment]
Hello bro this is test you can unmark as answer
